src/util/WorkflowsExplorer/task.ts

```diff
@@ -0,0 +1,20 @@
+import { Row } from '../../types';
+
+/**
+ * Row id might come as string or number. Make sure we have string
+ */
+export function getTaskId(task: Row): string {
```

Comment:
The Object oriented programming pattern would be to make this a method of the Row class. WDYT?

Reply:
Yes I agree. This is inherited from Metaflow and I haven't completely re-written the logic behind.

I feel this will be easier to figure out how to adapt the "legacy" code from Metaflow once we have a more or less clear vision of how we structure and fetch data from the backend/statefiles... 

Currently, the class "Attempt" makes the translation "Backend -> StateFile -> Metaflow Rows".. But it might be easier to understand if instead we go "Backend -> Metaflow Rows"... That would mean the translation occurs earlier, when the StateFiles are preprocessed and to the "backend" database